mem/basic.md

```diff
@@ -166,7 +166,7 @@ impl BasicMemory {
         // FIXME: This is *not* what rustc does, since from this it follows that
         // `offset(0)` is allowed on all non-null pointers which does not match
         // the LLVM IR rustc generates.
-        if len == 0 {
+        if len.is_zero() {
```

Comment:
I didn't want to implement `PartialEq<i32>` for `Size`, due to the bits v. bytes ambiguity.
Hence we need `ìs_zero`